_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -4597,7 +4597,6 @@
 /obj/item/bikehorn/rubberducky,
 /obj/effect/spawner/lootdrop/maintenance/two,
 /obj/item/clothing/gloves/color/fyellow,
-/obj/item/grenade/empgrenade,
```

Comment:
ehhhhhhhhhh...

okay, I guess? you really should add this to the general maint loot pool if it isn't in there already, though.